schemas/JSON/settings/settings.schema.0.2.json

```diff
@@ -294,21 +294,6 @@
           "type": "boolean",
           "default": false
         },
-        "configuration03": {
```

Comment:
I dream of the day we finally get around to versioning this file...
I wonder if we could publish it as an artifact for each release, and have the schema URI point to the release instead of the source code. Or at least point to a release branch